pkgs/top-level/aliases.nix

```
@@ -963,6 +963,10 @@ mapAliases ({
   unifi-poller = unpoller; # Added 2022-11-24
   unifiStable = unifi6; # Added 2020-12-28
   untrunc = untrunc-anthwlock; # Added 2021-02-01
+  urlview = throw ''
+    'urlview' has been dropped because it's unmaintained.
+    Consider switching to an alternative such as `pkgs.extract_url` or `pkgs.urlscan`.
+  ''; # Added 2023-12-14
```

```suggestion
  urlview = throw "'urlview' has been dropped because it's unmaintained. Consider switching to an alternative such as `pkgs.extract_url` or `pkgs.urlscan`."; # Added 2023-12-14
```
no idea if that would break the scripts scanning this file, so better safe than sorry

---

I think it's a very bad idea to do that, especially without a very clear warning sign (yes, a maintainer script is referenced, but it doesn't tell me how it deals with this file and I won't dig into that now ;-) )